Core/WumpusTesting/TriviaTesting.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.IO;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using WumpusCore.Trivia;
+
+namespace WumpusTesting
+{
+    [TestClass]
+    public class TriviaTesting
+    {
+        public Trivia makeTrivia()
+        {
+            string path = Path.GetFullPath("..\\..\\..\\WumpusCore\\Trivia\\Questions.json");
```

Comment:
We'll need to make sure to not use relative paths with the final game, but for testing this works

Reply:
its testing so its fine i think
its max's problem how to make it work in controller >:)